VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
The `RelativeLayout` is the last item in the `LinearLayout`, so it can be set to `match_parent`. On the other hand, there's definitely unneeded layers of layouts here, and the `RelativeLayout` could be brought to the top level under the `ScrollView`, and the header line could also be a child of it.

By the way, setting the `ScrollView` child's height to `match_parent` wouldn't make any difference, since it's already set to fill it's viewport.